frontend/src/Buildings/actionCreator.js

```diff
@@ -20,3 +23,35 @@ export function fetchBuildings() {
       .catch((error) => dispatch({ type: FETCH_BUILDINGS_FAILURE, payload: error }));
   };
 }
+
+let uid = 1;
+
+const getMockedBuilding = (type) => {
+  uid += 1;
+  return {
+    id: `temp-${uid}`,
```

Comment:
Since we didn't send the new building to the backend yet, we won't know the last id of the whole building. It's better to just create temp id for now.